jans-casa/app/src/main/resources/labels/admin.properties

```diff
@@ -51,7 +51,7 @@ adm.method_plugin_template={0} ({1})
 
 adm.methods_hint_noscripts={
 The list is empty! Please enable custom scripts (check the 
-<a href="#" target="_blank">docs</a>).
+<a href="https://docs.jans.io/v1.0.22/casa/administration/admin-console/#enabled-authentication-methods" target="_blank">docs</a>).
```

Comment:
Hi @KyleChaukccs 

Thanks for submitting this PR. 

As per our earlier conversation, use the link given below to point to the `head` instead of particular release.

```
https://docs.jans.io/head/casa/administration/admin-console/#enabled-authentication-methods
```

Reply:
@ossdhaval Sorry, my bad. I thought the "HEAD" handling method was not in place yet. I have done another commit accordingly. Please check. Many thanks.